src/App.css

```diff
@@ -0,0 +1,193 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+
+.App {
+  background-color: rgb(245, 246, 250);
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding: 30px 10px;
+}
+
+.nav-wrap {
+  background-color: white;
+  width: 100%;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-width: 0 0 2px 0;
+  border-color: rgb(228, 227, 227);
+  border-style: solid;
+}
+
+li {
+  list-style: none;
+  margin: 0;
+}
+
+.nav-wrap p {
+  margin-left: 62px;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+  font-family: "Montserrat", sans-serif;
+}
```

Comment:
You did great at adhering substancially to design guide. Kindly adjust the little alignment of the page heading with the cards as shown on the design. Also, within the cards, kindly align the title, category and comment vertically as shown on the design.

Your design:
![image](https://github.com/LDouglasOT/bookstores/assets/20554583/98a7ec98-9c50-43a8-995a-8c0fcd74b9f6)

Design Guide:
![image](https://github.com/LDouglasOT/bookstores/assets/20554583/3412fbb4-3fee-497d-bb99-bb7f5a85d857)